src/spec/validitygenerator.py

```diff
@@ -348,9 +348,9 @@ def createValidationLineForParameterIntroChunk(self, blockname, param, params, t
             if (lengths[0]) == 'null-terminated':
                 asciidoc += 'a null-terminated '
             elif (lengths[0]) == '1':
-                asciidoc += 'a pointer to '
+                asciidoc += 'a valid pointer to '
```

Comment:
Besides, you are using "insider knowledge".

The reader does not necessarily caries in the head, what the autovalidity generator will spit out and not under what circumstances.

I could equally say for e.g.:

> `renderPass` **must** be a <s>valid</s> VkRenderPass handle

"Ok, there would be no VU statement if `renderPass` can be nonsense value, ergo I do not need to say \'valid\' in the VU if it does have to be valid handle"